src/templates/resources/footer.json

```diff
@@ -1284,121 +1375,118 @@
             },
             {
-                "url": "/domicilio/hamburger",
-                "text": "Hamburger a domicilio"
+                "url": "/domicilio/vicino-a-me/kebab/",
+                "text": "Kebab a domicilio"
             },
             {
-                "url": "/domicilio/tipi-cucina",
+                "url": "/domicilio/vicino-a-me/",
                 "text": "Tutti i tipi di cucine"
             }
         ],
         "towns": "Città",
         "locationLinks": [
             {
-                "url": "/domicilio/palermo",
+                "url": "/domicilio/palermo/",
                 "text": "Palermo"
             },
             {
-                "url": "/domicilio/roma",
+                "url": "/domicilio/roma/",
                 "text": "Roma"
             },
             {
-                "url": "/domicilio/milano",
+                "url": "/domicilio/milano/",
                 "text": "Milano"
             },
             {
-                "url": "/domicilio/napoli",
+                "url": "/domicilio/napoli/",
                 "text": "Napoli"
             },
             {
-                "url": "/domicilio/Torino",
+                "url": "/domicilio/torino/",
                 "text": "Torino"
             },
             {
-                "url": "/domicilio/bologna",
+                "url": "/domicilio/bologna/",
                 "text": "Bologna"
             },
             {
-                "url": "/domicilio/citta",
+                "url": "/domicilio/citta/",
                 "text": "Tutte le città"
-            },
-            {
-                "url": "/domicilio/citta-offerte",
-                "text": "Offerte e Sconti"
             }
         ],
+        "brands": "Brands",
+        "brandLinks": [],
         "aboutUs": "Chi siamo",
         "aboutUsLinks": [
             {
-                "url": "/privacy-policy",
+                "url": "/privacy-policy/",
                 "text": "Termini e Condizioni"
             },
             {
-                "url": "/cookiespolicy",
+                "url": "/cookiespolicy/",
                 "text": "Cookie Policy"
             },
             {
-                "url": "/about",
+                "url": "/about/",
                 "text": "Informazioni su Just Eat"
             },
             {
-                "url": "/faq",
+                "url": "/faq/",
                 "text": "Domande frequenti"
             },
             {
-                "url": "/domicilio/miglior-prezzo-garantito",
+                "url": "/domicilio/miglior-prezzo-garantito/",
                 "text": "Miglior Prezzo Garantito"
             },
             {
-                "url": "/jobs",
+                "url": "/jobs/",
                 "text": "Lavora con noi"
             },
             {
-                "url": "https://www.justeat.it/domicilio/driver-partner",
+                "url": "https://www.justeat.it/domicilio/driver-partner/",
```

Comment:
just `/domicilio/driver-partner/`, don't need the full URL, although actually this redirects to `/informazioni/driver-partner` so you can use that instead.